File: python/CWE-190.py
def add_large_numbers(a, b):
    result = a + b
    return result

if __name__ == "__main__":
    max_int = 2**31 - 1  # Maximum value for a 32-bit signed integer
    a = max_int
    b = 1
    print(f"The result of adding {a} and {b} is: {add_large_numbers(a, b)}")

File: GO/CWE-190.go
package main

import "fmt"

func addLargeNumbers(a int32, b int32) int32 {
	return a + b
}

func main() {
	var maxInt int32 = 2147483647 // Maximum value for a 32-bit signed integer
	a := maxInt
	b := int32(1)
	fmt.Printf("The result of adding %d and %d is: %d\n", a, b, addLargeNumbers(a, b))
}
